R/utils-json.R

```diff
@@ -231,21 +238,12 @@ getApprovalDates <- function(data, chain_nm, approval){
 }
 
 #'@importFrom lubridate parse_date_time
-getTimeSeries <- function(ts, field, estimatedOnly = FALSE, isDV = FALSE){
+getTimeSeries <- function(ts, field, estimatedOnly = FALSE){
   y <- ts[[field]]$points[['value']]
   x <- ts[[field]]$points[['time']]
   
   if(!is.null(y) & !is.null(x)){
-    if(isDV) {
-      format <- "Ymd"
-    } else {
-      format <- "Ymd HMOS z"
-    }
-    
-    time <- parse_date_time(x,format, tz=ts$reportMetadata$timezone,quiet = TRUE)
-    if(isDV) { #want to render DVs at noon, convert to string to add noon and convert back to POSIXct
-      time <- time + hours(12)
-    }
+    time=flexibleTimeParse(x, ts$reportMetadata$timezone)
```

Comment:
ooo I like this! Might want to follow the current convention of `<-` instead of `=` though